api/app/Generators/CandidateProfileCsv.php

```diff
@@ -218,15 +233,26 @@ public function generate()
     private function generatePoolHeaders()
     {
 
-        Pool::with(['generalQuestions', 'poolSkills' => ['skill']])
+        Pool::with(['generalQuestions', 'screeningQuestions', 'poolSkills' => ['skill']])
             ->whereHas('poolCandidates', function ($query) {
                 $query->whereIn('id', $this->ids);
             })->chunk(100, function ($pools) {
                 foreach ($pools as $pool) {
                     if ($pool->generalQuestions->count() > 0) {
                         foreach ($pool->generalQuestions as $question) {
-                            $this->questionIds[] = $question->id;
-                            $this->generatedHeaders['general_questions'][] = $question->question[$this->lang];
+                            $this->generalQuestionIds[] = $question->id;
+                            $this->generatedHeaders['general_questions'][] = $this->lang == 'en' ?
+                            'General question: '.$question->question[$this->lang] :
+                            'Frenchy string: '.$question->question[$this->lang];
+                        }
+                    }
+
+                    if ($pool->screeningQuestions->count() > 0) {
+                        foreach ($pool->screeningQuestions as $question) {
+                            $this->screeningQuestionIds[] = $question->id;
+                            $this->generatedHeaders['screening_questions'][] = $this->lang == 'en' ?
+                            'Screening question: '.$question->question[$this->lang] :
+                            'Frenchy string: '.$question->question[$this->lang];
```

Comment:
What should be the string here?

Screening questions plural

![image](https://github.com/user-attachments/assets/ed0ad98a-bd74-4776-b7f9-034b8259a14a)

![image](https://github.com/user-attachments/assets/30ced7cb-bd1a-4fc4-bf05-5df462f73b36)

Reply:
We've a similar string below that reads differently as a heads up

![image](https://github.com/user-attachments/assets/fa514b56-27b0-4822-8db9-4788140795bc)

![image](https://github.com/user-attachments/assets/d33e5e42-47f0-4baa-81be-fabe47b83efb)

Reply:
Stick with singular, and the index is not necessary in the csv header.

Reply:
Let's go with `Question de sélection#. And while we're at it, fix that other to match: `Question de sélection {index}`.

Reply:
Thanks, I added/changed both in https://github.com/GCTC-NTGC/gc-digital-talent/pull/11093/commits/9b7967330169756d9ece240ec03f343fb488f8bb